src/site/content/en/blog/pwas-on-oculus-2/index.md

```diff
@@ -311,11 +294,30 @@ customize the PWA experience.
 </table>
 </div>
 
+### Packaging PWAs with PWABuilder
+
+Using [PWABuilder](https://pwabuilder.com) is the easiest and recommended way to package PWAs for
+Oculus Quest at the moment.
+
+PWABuilder is an [open source](https://github.com/pwa-builder), community guided project founded by
+Microsoft, that allows developers to package and sign their PWAs for publishing to various stores,
+including Microsoft Store, Google Play, App Store, and Oculus Store.
+
+Packaging PWAs with PWABuilder as easy as entering the URL of your PWA, selecting the platform
+(the **Meta Quest** in our case), entering/editing metadata for your app, and clicking **Generate**
+button.
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/fTX92hn8bIMpWVMoBUjZ.png", alt="Packaging PWAs for Oculus Quest with PWABuilder.", width="800", height="450" %}
```

Comment:
The image URL will be updated.

Reply:
Final URL `image/8WbTDNrhLsU0El80frMBGE4eMCD3/sWjB34HjwjatCMUz39Q3.png`.

Reply:
Changed.